animation_url/.gitignore

```diff
@@ -1,7 +1,7 @@
 .DS_Store
 node_modules
 /build
-/.svelte-kit
+/.svelte-kit/*
```

Comment:
@joewagner I tweaked this to allow the `.svelte-kit/tsconfig.jsont` that you added in another PR. guessing you wanted that checked into git?